cax-ap-cam/cax_ap_cam/const.py

```diff
@@ -0,0 +1,25 @@
+from siriuspy.envars import VACA_PREFIX as _vaca_prefix
+
+class Constants:
```

Comment:
I think this class can derivate from `siriuspy.csdev.Const`, and can be renamed to `Const` or `ImgProcConst`

Reply:
@anacso17 , I have added more commits to this module, trying to standardize it. it should be moved to siriuspy soon.